pages/about.html

```diff
@@ -4,9 +4,67 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Nunito:wght@400;700&display=swap" rel="stylesheet">
 </head>
 <body>
-  
+  <header>
+    <nav>
+        <!-- Main Navigation inside <header> -->
+        <ul class="menu">
+            <!-- <li id="logo"><a href="index.html">X</a></li> -->
+            <li><span>✨</span></li>
+            <li><a href="../index.html">Home</a></li>
+            <li><span>✨</span></li>
+            <!-- <li><a href="about.html">About</a></li> -->
+            <li><span>✨</span></li>
+            <li><a href="portfolio.html">Portfolio</a></li>
+            <li><span>✨</span></li>
+        </ul>
+    </nav>
+  </header>
+    <main class="lorem2">
+      <h1 class="about">A little bit about me.</h1>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+        Numquam in debitis laboriosam assumenda adipisci deleniti 
+        vitae ipsa maxime atque incidunt facere, libero nobis dolor 
+        totam! Cupiditate iure nulla dolore corporis.
+        <br>
+        <br>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+        Aut qui veritatis quis maiores nesciunt porro voluptatum, 
+        atque quisquam repudiandae nihil ratione ipsum iusto suscipit 
+        sunt ducimus quidem? Atque, non amet!
+        <br>
+        <br>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+        Quia eaque tempora magni id rem illo possimus ipsam repudiandae, 
+        minima quisquam officiis in praesentium temporibus aliquid nihil 
+        veritatis ea rerum velit.
+      </p>
+      <div class="responsive">
+        <img src="../images/picrewx.png" alt="cartoon rendition of xiomara">
+      </div>
```

Comment:
We could let `img` tag be responsible for the `responsive` class instead of surrounding it with a `div` tag